yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/call/YotiConstants.java

```diff
@@ -6,19 +6,20 @@ private YotiConstants() {}
 
     public static final String PROPERTY_YOTI_API_URL = "yoti.api.url";
 
-    private static final String DEFAULT_YOTI_HOST = "https://api.yoti.com";
+    public static final String DEFAULT_YOTI_HOST = "https://api.yoti.com";
     public static final String YOTI_API_PATH_PREFIX = "/api/v1";
     public static final String DEFAULT_YOTI_API_URL = DEFAULT_YOTI_HOST + YOTI_API_PATH_PREFIX;
 
     public static final String AUTH_KEY_HEADER = "X-Yoti-Auth-Key";
     public static final String DIGEST_HEADER = "X-Yoti-Auth-Digest";
     public static final String YOTI_SDK_HEADER = "X-Yoti-SDK";
+    public static final String YOTI_SDK_VERSION_HEADER = YOTI_SDK_HEADER + "-Version";
 
     public static final String CONTENT_TYPE = "Content-Type";
     public static final String CONTENT_TYPE_JSON = "application/json";
 
     public static final String JAVA = "Java";
-    public static final String BOUNCY_CASTLE_PROVIDER = "BC";
+    public static final String SDK_VERSION = JAVA + "-2.2.0";
```

Comment:
Why does it say `-2.2.0` is this a Java thing?

Reply:
We are using _major-minor-patch_ for our versioning.  Not likely we'll make much use of the patch release, but it's there if we need it.

So here the SDK version will be _Java-2.2.0_.

Reply:
Do we need to have `Java` in front of it? There's the separate `X-Yoti-SDK` header which will identify it as Java right? in [.NET](https://github.com/getyoti/yoti-dotnet-sdk/commit/7c22325549c581f4a7f075dc00e1235330a80a40) I just have it as the version without the identifier in front of it

Reply:
On the one hand, no we don't need it since we have the other header.

On the other hand, if we find we're looking at reports that are only based on one header value, not both, then knowing that it's **Java**-2.2.0 will be quite helpful.

Reply:
I thought these two headers `X-Yoti-SDK` and `X-Yoti-SDK-Version` always go together.